cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Scan.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.dataset.table;
+
+import co.cask.cdap.api.annotation.Beta;
+import co.cask.cdap.api.common.Bytes;
+import com.google.common.base.Objects;
+
+import javax.annotation.Nullable;
+
+/**
+ * Scan configuration for {@link Table}.
+ */
+@Beta
+public class Scan {
+  @Nullable
+  private final byte[] startRow;
```

Comment:
Nevermind, it's good that these can't be modified if we pass this along.  But if we add more options we may want to adopt a builder pattern here.